src/components/v2/chartDeploymentHistory/ChartDeploymentHistory.component.tsx

```diff
@@ -473,15 +473,10 @@ function ChartDeploymentHistory({
 
             if (installedAppInfo) {
                 requestPayload.installedAppId = installedAppInfo.installedAppId
-                requestPayload.installedAppVersionId = isExternal
-                    ? installedAppInfo.installedAppVersionId
-                    : selectedDeploymentHistory.installedAppVersionId
```

Comment:
vikram handshake required

Reply:
Done. Now in both Devtron charts & EA, installedAppVersionId will not be required in the rollback PUT request payload.
E.g. 
For the Devtron chart, the payload will be,
```
hAppId: ""
installedAppId: 535
version: 171
```

For EA, the payload will be,
```
hAppId: "1|default|sohel-new-2-test-chart"
installedAppId: 568
version: 5
```